lib/parsers/webhook_json_parser.rb

```diff
@@ -96,6 +98,24 @@ def deleted?
           false
         end
       end
+
+      def repo_name
+        case @vcs
+        when 'github'
+          @data['repository']['name']
+        end
+        # TODO: Bitbucket, Stash/Bitbucket Server, Gitlab, TFS
+        nil
```

Comment:
This should be in an ```else``` branch of the ```case``` statement. Same for ```repo_user``` below.

Reply:
That's what I started with, but rubocop hates me. `Redundant 'else'-clause`

Reply:
Does anyone know what the purpose of these two methods originally was?

Reply:
Sorry, it was obviously wrong before unless I added some `returns`.